src/WebSdk/Worker/Sdk/Sdk.targets

```diff
@@ -12,6 +12,7 @@ Copyright (c) .NET Foundation. All rights reserved.
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
   <Import Sdk="Microsoft.NET.Sdk" Project="Sdk.targets" />
+  <Import Condition="'$(WorkerSdkImportPublishSdk)' == 'true'" Sdk="Microsoft.NET.Sdk.Publish" Project="Sdk.targets" />
```

Comment:
Same as the props. It would be best to move this right after the project system sdk import.

Reply:
done, thank you!